src/contributors page/user_palette.js

```diff
@@ -1,12 +1,23 @@
 const user_palette = [
-{
-	background: "cornflowerblue",
-	color: "white"
-},
-{
-	background: "orange",
-	color: "black"
-}
-
-
-]
\ No newline at end of file
+	{
+		background: "cornflowerblue",
+		color: "white"
+	},
+	{
+		background: "orange",
+		color: "black"
+	},
+	{
+		background: "#e03131",
+		color: "red"
```

Comment:
Annu, here color will be applied to font color, and color can be only white or black.
so update the color for your objects
Also resolve the merge conflict, In case you need any help, Ask!

Reply:
My bad !
That was a misunderstanding for my part. I have corrected it